addons/account/models/company.py

```diff
@@ -262,10 +262,10 @@ class ResCompany(models.Model):
     )
     company_vat_placeholder = fields.Char(compute='_compute_company_vat_placeholder')
 
-    def get_next_batch_payment_communication(self):
+    def get_next_group_payment_communication(self):
```

Comment:
Better to avoid changing names of methods in fixes in stable.  (you can easily do it in master)  The reason is that people might have inherited the method in their custom modules and if we do not need to break, we avoid it.